config/finders/all_content.yml

```diff
@@ -64,5 +64,5 @@ details:
   - key: public_timestamp
     name: Updated (oldest)
 routes:
-- path: "/all-content"
+- path: "/search/all-content"
```

Comment:
The json route got removed at some point.  Live search uses this to update the results.  Any chance you could add this back in please?  (And for the guidance and regulation finder too please?).

I think those routes are still present in router which is why these finders still work, but they'll be broken when we republish these unless we add them back in.

Reply:
I added the JSON route for the all content finder.

Guidance and regulation appears to have a JSON route:
https://github.com/alphagov/rummager/pull/1459/files#diff-761566fda268c4381877f7ab723294cbR68

Reply:
Cool, just my eyes then 👍